src/app/core/components/navigation/style.scss

```diff
@@ -29,9 +29,15 @@ header {
   .km-header-logo {
     display: block;
     height: 30px;
-    margin-left: variables.$content-padding - 10px;
+    margin: 0 (variables.$content-padding - 10px);
+    padding-right: 160pt;
     width: 200px;
   }
+
+  .km-text {
+    font-size: variables.$font-size-subhead-lg;
+    font-weight: normal;
```

Comment:
add  `margin: 0 10px;` or margin: 0 0  0 10px; ( to only apply left) margin

this will add margin between `|` line and km-text

Reply:
you can even use `margin-left` only too 😄